app/views/declaration/additional_fiscal_references.scala.html

```diff
@@ -82,7 +82,7 @@
 @inputFields = {
     @components.fields.field_accessible_autocomplete(
         field = form("country"),
-        label = messages("declaration.additionalFiscalReferences.country"),
+        label = "declaration.additionalFiscalReferences.country",
```

Comment:
IMHO it is better for the label to consume the final message, not the key. This way you have more loose coupling between `field_accessible_autocomplete` and its clients.

Reply:
But this is how it was implemented before.

But the new specs will recognise if you use `messages(messages("messagesKey"))` which is a reason of this change

Reply:
Alright, didn't realise that. No problem then (at least for now).